pygmt/tests/test_xarray_backend.py

```diff
@@ -0,0 +1,57 @@
+"""
+Tests for xarray 'gmt' backend engine.
+"""
+
+import re
+
+import pytest
+import xarray as xr
+from pygmt.enums import GridRegistration, GridType
+from pygmt.exceptions import GMTInvalidInput
+
+
+def test_xarray_backend_gmt_read_grid():
+    """
+    Ensure that passing engine='gmt' to xarray.open_dataarray works for reading
+    NetCDF grids.
+    """
+    with xr.open_dataarray(
```

Comment:
Maybe also add one test for `xr.load_dataarray`?

Reply:
Added in commit c0ccd4b0cdbf1708ad7b6adca0750b0907a7005f